### PR TITLE
Python 3.9

### PR DESCRIPTION
**Pull request by mingdavidqi**
### Overview
upgrade Python and its dependencies from 3.6 to 3.9

### CODE REVIEW

File: examples/tensorflow/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,50 @@
...
+requests
+requests-oauthlib
+rsa
+scipy
+schema
+six
+tensorboard
+tensorboard-data-server
+tensorboard-plugin-wit
+tensorflow
```
Comment (rosskipp) [inline examples/tensorflow/requirements.txt:42]: do you need to specifiy a version of tensorflow? this model isn't working for me:
```
Traceback (most recent call last):
  File "cnn_mnist.py", line 13, in <module>
    tf.logging.set_verbosity(tf.logging.INFO)
AttributeError: module 'tensorflow' has no attribute 'logging'
```

**Review by rosskipp — CHANGES_REQUESTED** [review]

File: examples/article-summarizer/main.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import promote
-from schema import Schema
-
 import newspaper
+from schema import Schema
 from newspaper import Article
+import nltk
+nltk.download('punkt')
```
[the remark this answers is not in the document]
Comment (mingdavidqi) [reply examples/article-summarizer/main.py:6]: updated readme.md

File: examples/tensorflow/requirements.txt (hunk 1 above)
Comment (mingdavidqi) [reply examples/tensorflow/requirements.txt:42]: cnn_mnist.py only work with old version of tensoflow. I added a new file mnist.py for the upgraded version of tensorflow

Comment (rosskipp) [reply examples/tensorflow/requirements.txt:42]: okay - tested the mnist.py. file and had to add `matplotlib` and graphviz to get it to run. - can you add that to the requirements.txt and readme? you have to install graphviz from here: https://graphviz.gitlab.io/download/

Comment (mingdavidqi) [reply examples/tensorflow/requirements.txt:42]: The promote.sh will install graphviz. And matplotlib is install automatically with graphviz? I don't remember.  I will update readme and requirements

Comment (rosskipp) [reply examples/tensorflow/requirements.txt:42]: also maybe we should just remove the cnn_mnist file? or at least note in the README that you should use the other file

Comment (mingdavidqi) [reply examples/tensorflow/requirements.txt:43]: yeah, I just updated readme file to use mnist.py for training. leave cnn-mnist.py there and noticed it's for tensorflow < 2

File: examples/tensorflow/README.md [hunk 3]
```diff
@@ -9,22 +9,28 @@ tensorflow/
...
 ├── main.py
 └── requirements.txt
 ```
 
 ### Instructions
 
-Tensorflow only supports python 3.6-3.8 - so make sure you're on a version in that range.
+Tensorflow only supports python >= 3.6 - so make sure you're on a version in that range.
+
+Tensorflow 2 also need install graphviz which is included in file promote.sh. To manually install graphviz, please run:
```
Comment (rosskipp) [inline examples/tensorflow/README.md:22]: well - when you're training the model locally you also need it - so include the link to the install instructions (most people aren't using yum locally)

**Review by travisje — APPROVED** [review]

**Pull request merged**